src/components/event-log/event-log.stories.mdx

```diff
@@ -12,7 +12,43 @@ Displays a list of event details in chronological order. Designed to show the pr
 For a less specialized display of tabular data, consider using [Table](/docs/components-table--basic). For simple string/number pairs, consider [Dot Leader](/docs/components-dot-leader--list). To display a single or upcoming date, consider using [Calendar Date](/docs/components-calendar-date--basic) (optionally combined with [Media](/docs/objects-media--event-summary)).
 
 <Canvas>
-  <Story name="Example">{eventsDemoStory.bind({})}</Story>
+  <Story
+    name="Example"
+    parameters={{
+      docs: {
+        source: {
+          code: `{% embed '@cloudfour/components/event-log/event-log.twig' only %}
+  {% block content %}
+    {% for item in items %}
+      {% embed '@cloudfour/components/event-log/item.twig' with item only %}
+        {% block extra %}
+          {%- if item.links -%}
+            {% embed '@cloudfour/objects/list/list.twig' with {
+              class: 'o-list--inline'
+            } only %}
+              {% block content %}
+                {% for link in item.links %}
+                  <li>
+                    <a href="{{link.link}}">
+                      {{link.title}}
+                    </a>
+                  </li>
+                {% endfor %}
+              {% endblock %}
+            {% endembed %}
+          {%- endif -%}
+        {% endblock %}
+      {% endembed %}
+    {% endfor %}
+  {% endblock %}
+{% endembed %}
```

Comment:
@gerardo-rodriguez This looks like it might also be a candidate for loading the template contents via `raw-loader`. See my comment here: https://github.com/cloudfour/cloudfour.com-patterns/pull/1356#discussion_r661825591

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1357/commits/8998a90079f4c836af249a0e66459ac64a8406e5